src/it/java/com/google/checkstyle/test/chapter5naming/rule527localvariablenames/PatternVariableNameTest.java

```diff
@@ -38,22 +36,21 @@ protected String getPackageLocation() {
     @Test
     public void testPatternVariableName() throws Exception {
         final Configuration checkConfig = getModuleConfig("PatternVariableName");
-        final String format = checkConfig.getProperty("format");
-        final Map<String, String> messages = checkConfig.getMessages();
+        final String format = "^[a-z]([a-z0-9][a-zA-Z0-9]*)?$";
```

Comment:
I do not understand why we need to copy values from config to Tests.
We should load all modules related to covered part of style paragraph and confirm that violations are at expected places.
In most cases it is 1 check, but sometimes 2, or Check and filter.

Reply:
@romani This is an outdated PR based on our discussion. I waiting to understand your point on the other PR on the same issue before reworking this. Hence the blocked label.